Cython/Utility/ObjectHandling.c

```diff
@@ -295,13 +299,17 @@ static PyObject *__Pyx_PyObject_GetIndex(PyObject *obj, PyObject *index) {
 
     // Error handling code -- only manage OverflowError differently.
     if (PyErr_GivenExceptionMatches(runerr, PyExc_OverflowError)) {
+        __Pyx_TypeName index_type_name = __Pyx_PyType_GetName(Py_TYPE(index));
         PyErr_Clear();
-        PyErr_Format(PyExc_IndexError, "cannot fit '%.200s' into an index-sized integer", Py_TYPE(index)->tp_name);
+        PyErr_Format(PyExc_IndexError,
+            "cannot fit '%.200s' into an index-sized integer", index_type_name);
```

Comment:
Missing format macro.

Reply:
Good catch!

Reply:
Ah no, there is a test. But this is part of the code only reachable for `CYTHON_USE_TYPE_SLOTS` which is not true in the limited API. Oh well.